src/content/docs/query-your-data/explore-query-data/get-started/introduction-querying-new-relic-data.mdx

```diff
@@ -93,7 +93,7 @@ Regardless of your experience with New Relic, we'll help you discover, understan
 
 [Our platform](/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one) offers several experiences that don't require knowledge of NRQL or any query language. On the UI, go to the **Query your data** button, or click the **Browse data** dropdown, then select the data type (metrics, events, logs, and traces) you want to explore.
 
-* For [events](/docs/using-new-relic/data/understand-data/new-relic-data-types#event-data) and [metrics](/docs/using-new-relic/data/understand-data/new-relic-data-types#metrics), use the [data explorer](/docs/query-your-data/explore-query-data/data-explorer/introduction-data-explorer), an intuitive data navigator to create visualizations. From the explorer you can switch to the [query builder](/docs/chart-builder/use-chart-builder/choose-data/use-advanced-nrql-mode-specify-data) to see and refine your query.
+* For most of our data, you can use the [data explorer](/docs/query-your-data/explore-query-data/data-explorer/introduction-data-explorer), an intuitive data navigator for exploring data and creating visualizations. You can also use the [query builder](/docs/chart-builder/use-chart-builder/choose-data/use-advanced-nrql-mode-specify-data) to refine your query.
```

Comment:
"most of our" leaves me feeling a little uneasy. What data isn't part of "most of our"?